pom.xml

```diff
@@ -253,9 +253,19 @@
 			<version>2.16</version>
 		</dependency>
 		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpmime</artifactId>
+			<version>4.5.2</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.0</version>
+		</dependency>
```

Comment:
httpmime and commons-lang3 are imported into the test classes but do not exist with the latest selenium upgrade. When I changed the selenium update to the second latest version, these two dependencies were not required.